src/PostgreSQLCopyHelper.Test/TransactionalTestBase.cs

```diff
@@ -9,7 +9,7 @@ namespace PostgreSQLCopyHelper.Test
     public abstract class TransactionalTestBase
     {
         protected NpgsqlConnection connection;
-        private NpgsqlTransaction transaction;
+        protected NpgsqlTransaction transaction;
```

Comment:
Maybe I'm missing something but I'm not sure why this was needed. I don't think there would be negative side effects but I'm not sure about the intent here.

Reply:
Correct. Made it private.